config/etfuturum/blocksitems.cfg

```diff
@@ -0,0 +1,445 @@
+    B:enableHoney=false
+
+    #  [default: true]
+    B:enableLantern=true
+
+    # Invisible light blocks. Only has a selection box when held, right click to change light level. Otherwise functionally identical to air and can be replaced by placing blocks into it. Invisible, but can be seen when holding it in Creative mode. [default: true]
+    B:enableLightBlock=true
+
+    #  [default: true]
+    B:enableQuartzBricks=true
+
+    # Note: Also enables cracked and chiseled nether bricks as they use this ID too [default: true]
+    B:enableRedNetherBricks=true
+
+    #  [default: true]
+    B:enableSmoothQuartz=true
+
+    #  [default: true]
+    B:enableSmoothSandStone=true
+
+    #  [default: true]
+    B:enableSmoothStone=true
+
+    # Enables stripped log blocks [default: true]
+    B:enableStrippedLogs=true
+}
+
+
+##########################################################################################################
+# misc items
+#--------------------------------------------------------------------------------------------------------#
+# Items that don't fit in any other category.
+##########################################################################################################
+
+"misc items" {
+    #  [default: true]
+    B:enableBeetroot=true
+
+    #  [default: true]
+    B:enableIronNugget=true
+
+    #  [default: true]
+    B:enableLingeringPotions=true
+
+    #  [default: true]
+    B:enableMutton=true
+
+    #  [default: true]
+    B:enableNewDyes=true
+
+    # Appears in stronghold corridor and dungeon chests. [default: true]
+    B:enableOtherside=true
+
+    # Appears in Nether fortress chest loot. [default: true]
+    B:enablePigstep=true
+
+    # If true, vanilla and Et Futurum copper ores will drop raw ore items. [default: true]
+    B:enableRawOres=false
+
+    #  [default: true]
+    B:enableSuspiciousStew=true
+
+    #  [default: true]
+    B:enableTippedArrows=true
+
+    #  [default: true]
+    B:enableTotemUndying=true
+}
+
+
+##########################################################################################################
+# natural blocks
+#--------------------------------------------------------------------------------------------------------#
+# Blocks that can generate naturally in your world. Check world.cfg for generation values.
+##########################################################################################################
+
+"natural blocks" {
+    # Set how hard deepslate is, in case you like the look of deepslate but don't want to deal with the additional hardness. Regular stone is 1.5. [range: 0.0 ~ 100.0, default: 3.0]
+    S:deepslateHardness=2.3333333333
+
+    # Enables tinted glass, amethyst blocks, budding amethyst and amethyst crystals. Also enables the item too. [default: true]
+    B:enableAmethyst=false
+
+    # Enables bamboo wood and all of its wood subtypes, including the bamboo stalks themselves. [default: true]
+    B:enableBambooBlocks=true
```

Comment:
bamboo should probably be false.
1) we already have bamboo! (from bop)
2) the generation is so bad that testers just assumed it must be a bug

Reply:
Set to false for now. Ideally, we can have the bamboo block types without the bamboo world gen. I'll introduce some recipes for conversion and split this config in EFR between world gen and having access to the blocks.

Reply:
Roadhog is open to PRs or tickets - as long as it's config gated I see little reason he'd object

Reply:
https://github.com/Roadhog360/Et-Futurum-Requiem/pull/507

Reply:
> bamboo should probably be false.
> 1) we already have bamboo! (from bop)
> 2) the generation is so bad that testers just assumed it must be a bug

What's wrong with generation? Looks fine to me? Only issue I can see that the bamboo occasionally pokes through the leaves of trees but that doesn't seem to be fixable to my knowledge.